src/features/content-explorer/item-list/__tests__/ItemListIcon.test.js

```diff
@@ -33,33 +37,35 @@ describe('features/content-explorer/item-list/ItemListIcon', () => {
                 hasCollaborations: true,
                 isExternallyOwned: true,
             },
-        ].forEach(props => {
+            // externalFolder
+            {
+                type: 'folder',
+                hasCollaborations: false,
+                isExternallyOwned: true,
+            },
+        ].forEach(rowData => {
             test('should render correct folder icon', () => {
-                const wrapper = renderComponent(props);
-
-                expect(wrapper.find('FolderIcon').length).toBe(1);
-                expect(wrapper.prop('isCollab')).toEqual(props.hasCollaborations);
-                expect(wrapper.prop('isExternal')).toEqual(props.isExternallyOwned);
-                expect(wrapper.prop('title')).toBeTruthy();
+                const wrapper = renderComponent(rowData);
 
+                expect(wrapper.find('IconCell').length).toBe(1);
                 expect(wrapper).toMatchSnapshot();
             });
         });
 
         test('should render correct file icon', () => {
-            const extension = 'boxnote';
-            const wrapper = renderComponent({ type: 'file', extension });
+            const rowData = { type: 'file', extension: 'boxnote' };
+            const wrapper = renderComponent(rowData);
 
-            expect(wrapper.find('FileIcon').length).toBe(1);
-            expect(wrapper.prop('extension')).toEqual(extension);
-            expect(wrapper.prop('title')).toBeTruthy();
```

Comment:
do we not need to check the `title` anymore?

Reply:
It no longer has the `title` prop, only `rowData`. The `title` logic is now tested in [IconCell.test.ts](https://github.com/box/box-ui-elements/pull/3227/files/387dcd50fab1a887f2e2e72a1ade7c0b4130e37e#diff-b96c51318ab384cd29d726558833527aae42433a1620d61e045ec91763d1c170).